src/xcode/fastlane/Fastfile

```diff
@@ -141,7 +141,6 @@ platform :ios do
         output_directory: output_dir,
         result_bundle: true,
         skip_open_summary: true,
-        number_of_retries: 2,
```

Comment:
so we assume that all the tests will part in first try 😄

Reply:
Would have been nice if it worked but this way its at least not worse then our current CI xD